### DISCUSSION TITLE
【Bug】进度的Style从bar改为curve时，条目都不显示了

### DISCUSSION THREAD

**Discussion by ljs-fjmu-edu-cn** [MuiseDestiny/zotero-style#494] [Show and tell]
![屏幕截图 2023-07-20 101905](https://github.com/MuiseDestiny/zotero-style/assets/140003842/d6f3cc02-15d5-4d6e-8b9c-9cb6d3507730)

**Comment by MuiseDestiny** [comment]
![image](https://github.com/MuiseDestiny/zotero-style/assets/51939531/a361d21a-c450-422f-912f-a75b1dfbdf3c)

不支持curve

**Reply by MuiseDestiny** [reply]
请在config/编辑器里改回来，搜索zoterostyle.progressColumn

**Reply by ljs-fjmu-edu-cn** [reply]
是的，我自己摸索已经改回来了。就是为何我的是文本框不是下拉选择框？zotero 6 +zetero_style 2.6.7
![image](https://github.com/MuiseDestiny/zotero-style/assets/140003842/ad52aa39-e871-4b91-b01e-eb6a19995fed)

**Reply by MuiseDestiny** [reply]
升级到最新版就好啦

**Reply by vivelejrlee** [reply]
大佬，我也是用的zotero6.0.36+style2.6.7，但是Progress就是不显示，style选择是文本框，不是下拉选择框

**Reply by Uuuunaii** [reply]
> 是的，我自己摸索已经改回来了。就是为何我的是文本框不是下拉选择框？zotero 6 +zetero_style 2.6.7
> ![image](https://private-user-images.githubusercontent.com/140003842/255104902-ad52aa39-e871-4b91-b01e-eb6a19995fed.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MjE5NjUwNDIsIm5iZiI6MTcyMTk2NDc0MiwicGF0aCI6Ii8xNDAwMDM4NDIvMjU1MTA0OTAyLWFkNTJhYTM5LWU4NzEtNGI5MS1iMDFlLWViNmExOTk5NWZlZC5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjQwNzI2JTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI0MDcyNlQwMzMyMjJaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT1jMThiNzljOWZiY2RjYmNjZjA3OWYxODQyYjY1MjYzODIyMjlkY2UzMGU3M2U0ZDAwYjI4M2M1MTVmNmUyYTljJlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCZhY3Rvcl9pZD0wJmtleV9pZD0wJnJlcG9faWQ9MCJ9.WOiEC-h271UKz7h9bCj4OWEOW8DtQcmE6jtMdS2oY2A)

遇到了相同的情况，请问是怎么改回来的呢？求助！
